client/account-recovery/lost-password/lost-password-form/index.jsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class LostPasswordFormComponent extends Component {
+	constructor() {
+		super( ...arguments );
+
+		this.state = {
+			isSubmitting: false,
+			userLogin: '',
+		};
+	}
+
+	onSubmit = () => {
+		this.setState( { isSubmitting: true } );
+
+		//TODO: dispatch an event with userLogin and wait to here back
+	};
+
+	onUserLoginChanged = ( event ) => {
+		this.setState( { userLogin: event.target.value } );
+	};
+
+	render() {
+		const { translate } = this.props;
+		const { isSubmitting, userLogin } = this.state;
+		const isPrimaryButtonDisabled = ! userLogin || isSubmitting;
+
+		return (
+			<div>
+				<h2 className="lost-password-form__title">
+					{ translate( 'Lost your password' ) }
+				</h2>
+
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/8955/commits/c9ecb42ffd26cb51a0f6e68fd1f32fde092ff7e9